### PR TITLE
Make failed login message translation friendly

### PR DESCRIPTION
**Pull request by fabiopaiva**

### CODE REVIEW

File: src/ZfcUser/Controller/UserController.php [hunk 1]
```diff
@@ -111,7 +110,14 @@ public function loginAction()
         $form->setData($request->getPost());
 
         if (!$form->isValid()) {
-            $this->flashMessenger()->setNamespace('zfcuser-login-form')->addMessage($this->failedLoginMessage);
+            if($this->getServiceLocator()->has('translator')){
+                $translator = $this->getServiceLocator()->get('translator');
+                $failedLoginMessage = $translator->translate($this->failedLoginMessage);
+            }
```
Comment (stijnhau) [inline src/ZfcUser/Controller/UserController.php]: Should be one 1 line with a space before and after else for better readability

**Comment by Danielss89** [comment]
Why not do this in the view?

**Comment by fabiopaiva** [comment]
@Danielss89 
https://github.com/ZF-Commons/ZfcUser/blob/1.x/src/ZfcUser/Controller/UserController.php#L48
https://github.com/ZF-Commons/ZfcUser/blob/1.x/view/zfc-user/user/login.phtml#L13

**Comment by stijnhau** [comment]
indead i aslo wouldn't knwo how to that in the view

**Comment by fabiopaiva** [comment]
@stijnhau check now

**Comment by Danielss89** [comment]
Take a look at how it's done in change-password. I would suggest doing it the same way here.

**Comment by Danielss89** [comment]
Closing this for now. This should be done in the view.

File: src/ZfcUser/Controller/UserController.php [hunk 2]
```diff
@@ -111,7 +110,13 @@ public function loginAction()
         $form->setData($request->getPost());
 
         if (!$form->isValid()) {
-            $this->flashMessenger()->setNamespace('zfcuser-login-form')->addMessage($this->failedLoginMessage);
+            if ($this->getServiceLocator()->has('translator')) {
```
Comment (Danielss89) [inline src/ZfcUser/Controller/UserController.php]: Translater should be injected. It's not good to have a dependency on service-locator

**Pull request closed without merging**